tests/test_dm_control.py

```diff
@@ -103,6 +104,36 @@ def test_seeding(env_id):
     env_2.close()
 
 
+@pytest.mark.skip(
+    reason="Fatal Python error: Segmentation fault (with or without EzPickle"
```

Comment:
Do you know the source of this? Mujoco is the probable cause which we can probably get solved

Reply:
Looks like it's to do with pluggy?
```
/Users/elliottower/anaconda3/envs/shimmy/bin/python /Applications/PyCharm.app/Contents/plugins/python/helpers/pycharm/_jb_pytest_runner.py --target test_dm_control.py::test_pickle 
Testing started at 1:17 PM ...
Launching pytest with arguments test_dm_control.py::test_pickle --no-header --no-summary -q in /Users/elliottower/Documents/GitHub/Shimmy/tests

============================= test session starts ==============================
collecting ... collected 85 items

test_dm_control.py::test_pickle[dm_control/acrobot-swingup-v0] Fatal Python error: Segmentation fault

Current thread 0x000000011149c600 (most recent call first):
  File "/Users/elliottower/Documents/GitHub/Shimmy/tests/test_dm_control.py", line 114 in test_pickle
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/_pytest/python.py", line 192 in pytest_pyfunc_call
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_callers.py", line 39 in _multicall
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_manager.py", line 80 in _hookexec
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_hooks.py", line 265 in __call__
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/_pytest/python.py", line 1761 in runtest
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/_pytest/runner.py", line 166 in pytest_runtest_call
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_callers.py", line 39 in _multicall
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_manager.py", line 80 in _hookexec
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_hooks.py", line 265 in __call__
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/_pytest/runner.py", line 259 in <lambda>
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/_pytest/runner.py", line 338 in from_call
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/_pytest/runner.py", line 258 in call_runtest_hook
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/_pytest/runner.py", line 219 in call_and_report
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/_pytest/runner.py", line 130 in runtestprotocol
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/_pytest/runner.py", line 111 in pytest_runtest_protocol
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_callers.py", line 39 in _multicall
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_manager.py", line 80 in _hookexec
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_hooks.py", line 265 in __call__
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/_pytest/main.py", line 347 in pytest_runtestloop
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_callers.py", line 39 in _multicall
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_manager.py", line 80 in _hookexec
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_hooks.py", line 265 in __call__
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/_pytest/main.py", line 322 in _main
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/_pytest/main.py", line 268 in wrap_session
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/_pytest/main.py", line 315 in pytest_cmdline_main
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_callers.py", line 39 in _multicall
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_manager.py", line 80 in _hookexec
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/pluggy/_hooks.py", line 265 in __call__
  File "/Users/elliottower/anaconda3/envs/shimmy/lib/python3.9/site-packages/_pytest/config/__init__.py", line 164 in main
  File "/Applications/PyCharm.app/Contents/plugins/python/helpers/pycharm/_jb_pytest_runner.py", line 51 in <module>

Process finished with exit code 139 (interrupted by signal 11: SIGSEGV)
```